src/compiler/checker.ts

```diff
@@ -26473,7 +26605,8 @@ namespace ts {
                         return (<StringLiteral>expr).text;
                     case SyntaxKind.NumericLiteral:
                         checkGrammarNumericLiteral(<NumericLiteral>expr);
-                        return +(<NumericLiteral>expr).text;
+                        const literalValue = getNumericLiteralValue(<NumericLiteral>expr);
+                        return typeof literalValue === "number" ? literalValue : undefined; // don't evaluate bigint

```

Comment:
Need to handle evaluation of bigInt as enumMember value and a test. Cannot return undefined.. Need to add test for `const enums` with member initializers and operators.